tests/requirements-test.txt

```diff
@@ -1,6 +1,8 @@
 versioneer
-pytest
+# pytest v3.0 (from 19.08.2016) is no longer compatible with Python 3.2.x
+pytest<3.0 ; python_version >= '3.2' and python_version < '3.3'
+pytest ; python_version < '3.0' and python_version >= '3.3'
 pytest-capturelog
 pytest-timeout
 pytest-xdist
-tox
\ No newline at end of file
+tox
```

Comment:
Ok, I did this commit using online GitHub editor and it seems to add this \n at the end. Just look at 3rd commit here - no changes detected, but I removed it.